app/controllers/passwords_controller.rb

```diff
@@ -18,10 +18,10 @@ def create
   end
 
   def update
-    user = User.find_by(reset_token: params[:token])
+    user = User.find_by(token: params[:token])
     user.password = params[:password]
-    user.reset_token = nil
-    user.reset_token_expiration = nil
+    user.token = nil
+    user.token_expiration = nil
     user.save!
```

Comment:
Indeed, I wonder why I did it this way. Thanks for the tip, I'll refactor it. 

I've just read up on the safe navigation operator, which is a really nice new feature of Ruby 2.3. Thanks!